tests/pdarray_creation_test.py

```diff
@@ -323,15 +324,20 @@ def test_random_strings_uniform(self):
         self.assertEqual('type of argument "size" must be int; got str instead', 
                          cm.exception.args[0])              
 
-    def test_random_strings_uniform_with_seed(self):
-        pda = ak.random_strings_uniform(minlen=1, maxlen=5, seed=1, size=10)
- 
-        self.assertTrue((ak.array(['TVKJ', 'EWAB', 'CO', 'HFMD', 'U', 'MMGT', 
+    def test_random_strings_uniform_with_seed(self):        
+        try:
+            locales = int(os.environ['ARKOUDA_NUMLOCALES'])
```

Comment:
You should be able to get this with `locales = ak.get_config()['numLocales']`, which is used in some other tests and would avoid needing the env var and the try/except block here.